papers/Raku_Steering_Council_Code.md

```diff
@@ -57,11 +42,11 @@ The Steering Council shall work to:
 compilers and its ecosystem.
 * Make contributing and learning it as accessible, inclusive, and sustainable as
 possible.
-* Formalize and maintain the relationship between the core team and
-The Raku Foundation<sup id="a1">[1](#f1)</sup>,
-* Establish appropriate decision-making processes for problem solving
+* Formalize and maintain the relationship between the Core Team and
+The Raku Foundation.
+* Establish appropriate decision-making processes for Problem Solving
 proposals,
-* Seek consensus among contributors and the core team before acting in
+* Seek consensus among contributors and the Core Team before acting in
 a formal capacity,
 * Act as a "court of final appeal" for decisions where all other
```

Comment:
Don't know about the word "court" or anything implying judgement. Lesser words such as, well, council or even committee or task group would be much better.